google/cloud/ndb/_legacy_entity_pb.py

```diff
@@ -0,0 +1,762 @@
+#!/usr/bin/env python
+#
+# Copyright 2007 Google Inc.
```

Comment:
Update this to use the preamble used in our other source? @andrewsg @crwilcox  When should copyright year be updated?